app/assets/javascripts/base.js.erb

```diff
@@ -724,6 +724,7 @@ var instantClick
   });
 
   Honeybadger.beforeNotify(function(notice) {
+    if (!notice || typeof notice.message !== 'string') return true
```

Comment:
I've added a guard for the case where this function receives a non-string argument.